sklearn/ensemble/tests/test_bagging.py

```diff
@@ -288,7 +288,8 @@ def test_probability():
                                   np.exp(ensemble.predict_log_proba(X_test)))
 
         # Degenerate case, where some classes are missing
-        ensemble = BaggingClassifier(base_estimator=LogisticRegression(),
+        ensemble = BaggingClassifier(base_estimator=LogisticRegression(
+                                          solver='lbfgs'),
```

Comment:
That's a good point. Perhaps a better solution would be to change the value of `support_sample_weight`  to `False` in `_parallel_build_estimators` in bagging.py if `liblinear` is used.